kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue

```diff
@@ -2,7 +2,7 @@
 
   <div class="wrapper">
     <h1 class="section-header" :style="{ color: `${$themeTokens.annotation}` }">
-      {{ activeSection.section_title }}
```

Comment:
This is the line of code where Radina's replacements sidepanel was failing... not sure if my changes would help avoid that or not though.